tests/exceptions/nested_wrapping.py

```diff
@@ -28,5 +28,6 @@ def a(x):
         f(0)
     except ZeroDivisionError:
         logger.exception("")
+        raise
 except Exception:
     logger.exception("")
```

Comment:
The outer exception is not covered, but tests fail with this.
I think that might indicate some missing test (i.e. you want to also test the exception being re-raised / another exception being caught in the outer block).

Reply:
Actually, the whole nested `try` blocks could be removed. This is not testing anything useful. Previously these test cases were generated automatically through some kind of template strings. This is no longer the case, I did not realize I could remove it.